.ruby-version

```diff
@@ -1 +1 @@
-jruby-9.4.9.0
+jruby-9.4.3.0
```

Comment:
This looks like an unintended change, perhaps a bad merge conflict resolution? https://github.com/elastic/logstash/pull/16818/commits/ff5f491ce3e9cc5ae92c9d0864debb1b3aa57b77

Reply:
Oh thx ❤️